services/QuillLMS/app/queries/scorebook/query.rb

```diff
@@ -68,11 +74,15 @@ def self.run(classroom_id, current_page=1, unit_id=nil, begin_date=nil, end_date
           unit_activities.publish_date,
           unit_activities.due_date,
           unit_activities.created_at,
-          unit_owner.time_zone
+          unit_owner.time_zone,
+          psi.id,
+          upsi.id,
+          upsi.status
```

Comment:
Pretty sure that you don't need this `status` in `GROUP BY`.  (We haven't really been taking advantage of this, but if you `GROUP BY <table>.id` then PSQL recognizes that you're grouping by every column in that table (since ID is the most unique value in the table).  So this  means we really don't need `students.name` or `activity.name` in this `GROUP BY` either, but I don't think you need to go back and clean those up.